src/main/java/net/minestom/server/crypto/SignatureValidator.java

```diff
@@ -0,0 +1,94 @@
+package net.minestom.server.crypto;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.serializer.gson.GsonComponentSerializer;
+import net.minestom.server.entity.Player;
+import net.minestom.server.utils.crypto.KeyUtils;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+import java.security.*;
```

Comment:
Star imports no good

Reply:
What's wrong with them? IntelliJ said they are fine.

Reply:
The problem with star imports is that they import everything from that package. Therefore, not only is it difficult to see exactly what's being used, but it's also annoying if you want to use something else that's got the same name as one of these things. There are other reasons that I can't really recall that are better than this, but it's also [against Google's style guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) too.

Reply:
Also, there is another instance in here when IntelliJ corrected a star import to a separated import, which again makes this inconsistent.